src/components/Terminal/index.tsx

```diff
@@ -162,7 +162,6 @@ export class Terminal extends React.Component<ThemedTerminalProps, {}> {
 				});
 			}
 			this.resize();
-			// @ts-expect-error this was added to the window types on TS 4.2
```

Comment:
You need to add this back.
I'm guessing your VSCode is not using the TS version of the workspace, and that's why it shows up as an error.
> src/components/Terminal/index.tsx(165,40): error TS2339: Property 'ResizeObserver' does not exist on type 'Window & typeof globalThis'.